src/core/coap/coap_secure.hpp

```diff
@@ -148,6 +168,17 @@ class CoapSecure : public CoapBase
      */
     MeshCoP::Dtls &GetDtls(void);
 
+    /**
+     * This method sets the connected callback to indicate, when
+     * a Client connect to the CoAP Secure server.
+     *
+     * @param[in]  aMessageInfo  A reference to an address of the peer.
+     * @param[in]  aCallback     A pointer to a function that will be called once DTLS connection is
```

Comment:
nit: remove Line 175 and add missing `aContext`?

Reply:
Of course.
Done